package.json

```diff
@@ -298,6 +298,39 @@
         "language": "sas",
         "path": "./snippets/function-snippets.json"
       }
+    ],
+    "viewsContainers": {
+      "activitybar": [
+        {
+          "id": "sas-view-container",
+          "title": "SAS",
+          "icon": "icons/sas-svg.svg"
+        }
+      ]
+    },
+    "views": {
+      "sas-view-container": [
+        {
+          "id": "sas-content-pane",
+          "name": "SAS Content"
+        },
+        {
+          "id": "sas-libraries-pane",
+          "name": "Libraries"
+        }
+      ]
+    },
+    "viewsWelcome": [
+      {
+        "view": "sas-content-pane",
+        "contents": "Please [login](command:SAS.authenticate) to view SAS content.",
+        "when": "!SAS.session.outputHtml"
```

Comment:
@scnwwu Do you know if there is a way to detect if a user has a current valid session? This doesn't quite work. The general idea here is that we want to encourage users to login to see SAS Content if they aren't logged in.

Reply:
I think the extension should maintain a login status. (Implement a https://code.visualstudio.com/api/references/vscode-api#AuthenticationProvider). That will be shared across various functionalities.